### DISCUSSION TITLE
Control over header image

### DISCUSSION THREAD

**Discussion by miaucl** [gohugo-ananke/ananke#729] [3 - Ideas]
Currently, the header image for a post is `bg-top`, but it would be nice the control this behavior to lets say `top|center|bottom`, as often the interesting part of an image is in the center.

Relevant code bit: https://github.com/theNewDynamic/gohugo-theme-ananke/blob/33178c0d498edb18634bfe0f906ca191796ce307/layouts/partials/page-header.html#L4

**Comment by miaucl** [comment]
I would imagine something like that and also tested it as such with values `top|center|bottom` to correctly align the image: https://github.com/theNewDynamic/gohugo-theme-ananke/commit/5d7339037b07f87bf56b9ea9d06f9ae2871817d3

I know, the field `featured_image_alignment` is not ideal and one would probably want to rethink the `params` for the `featured_image` and maybe even add more control to it.

Before creating a PR for it (as mentioned in the PR notes ^^), let me know what you think of it.

**Reply by davidsneighbour** [reply]
I am all in with that, but I would suggest to put the full bg-top into the default and let users define the class fully ([bg-center, bg-bottom, etc.](https://github.com/tachyons-css/tachyons-background-position)). This way they can define their own additional classes (like hover treatment, etc). Call it, for instance, `featured_image_classes`. There is no larger setup for page options yet, that can be down at a later point.

An identical set up in `site-header.html` could receive the same treatment.

**Reply by miaucl** [reply]
There you go, I tried to comply as good as possible with existing conventions: https://github.com/theNewDynamic/gohugo-theme-ananke/pull/731

Not sure why the build fails, though.

**Reply by davidsneighbour** [reply]
I have a release planned for next Sunday (Oct 20th) where this will be included.

**Comment by djotaku** [comment]
@miaucl would this be adjustable per .md file? (Like I might want top for the main blog and bottom for another post, etc)

**Reply by miaucl** [reply]
Yes, have a look at the `exampleSite` or the PR: https://github.com/theNewDynamic/gohugo-theme-ananke/pull/731
